static/js/components/JobDetails.jsx

```diff
@@ -0,0 +1,168 @@
+var CronRow = React.createClass({
+  getInitialState: function() {
+    return {clicked: false}
```

Comment:
Is "clicked" representing a state or an action? At first glance, clicked feels like an action. I don't have a strong opinion on this.